beamer/tests/agent/unit/test_handlers.py

```diff
@@ -289,3 +298,76 @@ def test_maybe_claim_l1_as_challenger(
         assert mocked_withdraw.called
     else:
         assert not mocked_withdraw.called
+
+
+@patch("beamer.metrics")
+def test_handling_claim_during_sync(_mocked_metrics):
+    """
+    Tests the problem encountered in https://github.com/beamer-bridge/beamer/issues/782
+    """
+    context, config = make_context()
+    # Assert that ACCOUNT.address is the agent's address
+    # Is used as an assurance that in case ACCOUNT is changed
+    assert config.account.address == ACCOUNT.address
```

Comment:
The comment looks a bit strange, I'm not sure I understand what is being checked here. What purpose does `ACCOUNT` have and how does that relate to the agent?

Reply:
I just copied this from another test, but I'm fine with removing it.